examples/post-process/index.html

```diff
@@ -0,0 +1,65 @@
+<html>
+
+<head>    
+     <script type="importmap">
+        {
+          "imports": {
+            "aframe": "../../dist/aframe-master.module.min.js",
+            "three": "https://cdn.jsdelivr.net/npm/super-three@0.172.0/build/three.module.js",
+            "three/addons/": "https://cdn.jsdelivr.net/npm/super-three@0.172.0/examples/jsm/"
```

Comment:
It should be
```
          "three": "../../../super-three-package/build/three.module.js",
          "three/addons/": "../../../super-three-package/examples/jsm/",
```
like https://github.com/aframevr/aframe/blob/master/examples/boilerplate/importmap/index.html
so it uses the three version from node_modules.